tests/acceptance/change.spec.js

```diff
@@ -0,0 +1,319 @@
+'use strict';
+
+var expect = require('chai').expect;
+var path = require('path');
+var change = require('../../addon/ng2/utilities/change');
+var mockFs = require('mock-fs');
+var existsSync = require('exists-sync');
+var fsPromise = require('../../addon/ng2/utilities/fs-promise');
+
+describe('Change', () => {
+  var sourcePath;
+
+  describe('Invalid Path', () => {
```

Comment:
describe is used to group tests together, and to share common setup via beforeEach/afterEach (which run after each 'it' test case).

Each test should be an `it`, not a `describe` with a `beforeEach`, `afterEach`, and a single test. The `beforeEach`es here are duplicated.